src/shared/util/split-events.js

```diff
@@ -1,20 +1,43 @@
 import dateFns from 'date-fns';
 
-export function splitEvents(events, timeline, { reverse = false } = {}) {
+export function splitEvents({
+  events,              // array of events
+  timeline,            // one of 'past', 'today', 'future'
+  reverse = false,     // optionally reverse final list of events
+  todayDateTime = null,  // iso string representing today date
+}) {
+  const today = todayDateTime || new Date();
```

Comment:
...or not. Seems to be working fine also when assigned as a default value and destructured. I'll fix it.